src/zenml/models/filter_models.py

```diff
@@ -303,6 +317,26 @@ def list_of_filters(self) -> List[Filter]:
             {key: getattr(self, key) for key in self.__fields__}
         )
 
+    @property
+    def sorting_params(self) -> Tuple[str, SorterOps]:
+        """Converts the class variables into a list of usable Filter Models."""
+        column = self.sort_by
+        # The default sorting operand is asc
+        operator = 'asc'
+
+        # Check if user explicitly set an operand
+        split_value = self.sort_by.split(":", 1)
+        if (
+                len(split_value) == 2
+        ):
+            column = split_value[1]
+
+            if split_value[0] in SorterOps.values():
```

Comment:
Isn't this already covered in the validator?

Reply:
But the `if` condition can be removed right? It already validates that `split_value[0]` is in `SorterOps.values()`

Reply:
You are of course correct, i removed it